### PR TITLE
Feat/boilerplate enhancements

### PR DESCRIPTION
**Pull request by almeida-shrutika**
Changes:
- Update App title
- Add locales files for rte labels
- Remove 0 properties from rte css file
- Remove @emotion/core package from rte
- Update the folder structure for /src folder
- Add locales fils for the text labels
- Add an example data.json file for playwright tests
- Replace jotai library with context API

### CODE REVIEW

**Review by surajcontentstack — CHANGES_REQUESTED** [review]

File: src/common/hooks/useAppSdk.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ */
+import { MarketplaceContext, MarketplaceContextType } from "../contexts/marketplaceContext";
+import { useContext } from "react";
+
+/**
+ * Getter and setter for appSdk instance.
+ * To be used during Sdk initialisation
+ */
+export const useAppSdk = () => {
+  const { appSdk } = useContext(MarketplaceContext) as MarketplaceContextType;
```
Comment (surajcontentstack) [inline src/common/hooks/useAppSdk.tsx:13]: REname context to "MarketplaceAppContext

File: src/common/hooks/useInstallationData.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import {
+  AppConfigurationExtensionContext,
+  AppConfigurationExtensionContextType,
+} from "../contexts/appConfigurationExtensionContext";
+
+/**
+ * Getter & Setter for installation data
+ */
+export const useInstallationData = () => {
+  const { installationData, loading } = useContext(
```
Comment (surajcontentstack) [inline src/common/hooks/useInstallationData.tsx:11]: setter function is missing

**Review by surajcontentstack — CHANGES_REQUESTED** [review]

File: src/containers/CustomField/CustomField.test.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          customField={customField.customField}
+          setFieldData={customField.setFieldData}
+          loading={customField.loading}>
+          {children}
+        </CustomFieldTestProvider>
+      </TestProvider>
+    ),
+  });
+
+  //   await waitFor(() => {
```
Comment (surajcontentstack) [inline src/containers/CustomField/CustomField.test.tsx:36]: Need test cases

File: src/setupTests.ts [hunk 4]
```diff
@@ -5,5 +5,5 @@
 import "@testing-library/jest-dom";
 
 // mock the third party modules
-jest.mock("mixpanel-browser");
-jest.mock("trackjs");
+// jest.mock("mixpanel-browser");
```
Comment (surajcontentstack) [inline src/setupTests.ts:8]: Remove all commented code

**Review by surajcontentstack — CHANGES_REQUESTED** [review]

**Review by surajcontentstack — CHANGES_REQUESTED** [review]

**Review by surajcontentstack — APPROVED** [review]
Looks good to me

**Review by abhishek305 — APPROVED** [review]

**Pull request merged**